app/styles/app.sass

```diff
@@ -1559,6 +1560,8 @@ ul.risk-table
     .remove
       font-size: 14px
       color: $critical
+    input[type="checkbox"]
+      margin-top: 2px
```

Comment:
It is better to avoid such custom styles, thats why I used the existing class `vertical-align-center` here: https://github.com/appknox/irene/pull/679/files#diff-b12496c16d96d2b4ea569f7ab1034b7eL13